src/interface/whitelist/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const router = express.Router();
+
+const {
+    asyncHandler,
+    asyncHandlerArray,
+} = require('../../infra/asyncHandler');
+
+const whitelistAddress = require('./address');
+
+router.post('/address', asyncHandlerArray(whitelistAddress));
```

Comment:
Here can you add the support for client / secret API key (passed in the headers)? also add caller as extra key param. So that we can filter it later on or know who called the last batch.

Reply:
sure, adding it.